shell/utils/dynamic-content/info.ts

```diff
@@ -131,6 +156,10 @@ export class SystemInfoProvider {
     const screenSize = `${ window.screen?.width || '?' }x${ window.screen?.height || '?' }`;
     const browserSize = `${ window.innerWidth }x${ window.innerHeight }`;
 
+    Object.entries(ffs).forEach(([id, ff]) => {
+      ff.value = getters['features/get'](id);
```

Comment:
wrap in try catch given https://github.com/rancher/dashboard/blob/master/shell/store/features.js#L56